src/universal_transfer_operator/data_providers/database/google/bigquery.py

```diff
@@ -1,19 +1,67 @@
 from __future__ import annotations
 
+import time
+from typing import Any, Callable, Mapping
+
+import attr
 import pandas as pd
 from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook
+from airflow.providers.google.cloud.hooks.bigquery_dts import BiqQueryDataTransferServiceHook
 from google.api_core.exceptions import (
+    ClientError,
+    Conflict,
+    Forbidden,
+    GoogleAPIError,
+    InvalidArgument,
     NotFound as GoogleNotFound,
+    ResourceExhausted,
+    RetryError,
+    ServerError,
+    ServiceUnavailable,
+    TooManyRequests,
+    Unauthorized,
+    Unknown,
+)
+from google.cloud import bigquery, bigquery_datatransfer  # type: ignore
+from google.cloud.bigquery_datatransfer_v1.types import (
+    StartManualTransferRunsResponse,
+    TransferConfig,
+    TransferState,
 )
+from google.protobuf import timestamp_pb2  # type: ignore
+from google.protobuf.struct_pb2 import Struct  # type: ignore
+from google.resumable_media import InvalidResponse
 from sqlalchemy import create_engine
 from sqlalchemy.engine.base import Engine
+from tenacity import retry, stop_after_attempt
 
-from universal_transfer_operator.constants import DEFAULT_CHUNK_SIZE, LoadExistStrategy
+from universal_transfer_operator.constants import (
+    DEFAULT_CHUNK_SIZE,
+    FileLocation,
+    FileType,
+    LoadExistStrategy,
+)
 from universal_transfer_operator.data_providers.database.base import DatabaseDataProvider
+from universal_transfer_operator.datasets.file.base import File
 from universal_transfer_operator.datasets.table import Metadata, Table
+from universal_transfer_operator.exceptions import DatabaseCustomError
 from universal_transfer_operator.settings import BIGQUERY_SCHEMA, BIGQUERY_SCHEMA_LOCATION
 from universal_transfer_operator.universal_transfer_operator import TransferIntegrationOptions
 
+DEFAULT_CONN_ID = BigQueryHook.default_conn_name
+NATIVE_PATHS_SUPPORTED_FILE_TYPES = {
+    FileType.CSV: "CSV",
+    FileType.NDJSON: "NEWLINE_DELIMITED_JSON",
+    FileType.PARQUET: "PARQUET",
+}
+BIGQUERY_WRITE_DISPOSITION = {"replace": "WRITE_TRUNCATE", "append": "WRITE_APPEND"}
+
+
+@attr.define
+class BigqueryOptions(TransferIntegrationOptions):
+    s3_transfer_parameters: dict = attr.field(factory=dict)
+    gs_transfer_parameters: dict = attr.field(factory=dict)
```

Comment:
```suggestion
    gcs_transfer_parameters: dict = attr.field(factory=dict)
```
for consistency since I see we mention gcs_bucket in above file and not gc_bucket. Or can we rename gcs_bucket to gs_bucket above?

Reply:
I have changed it to `gs` everywhere as suggested: [Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/66/commits/502a76a396e543847c2327e5e51861abc4854f12)